docs/_reporting/power-bi/setup.md

```diff
@@ -68,12 +68,24 @@ The FinOps toolkit Power BI reports include pre-configured visuals, but are not
      <blockquote class="warning" markdown="1">
        _[Enable incremental refresh](https://learn.microsoft.com/power-bi/connect-data/incremental-refresh-configure#define-policy) to load more than $5M of raw cost details. Power BI reports can only support $2-5M of data when incremental refresh is not enabled. After incremental refresh is enabled, they can support $2-5M/month for a total of ~$65M in raw cost details._
      </blockquote>
-4. Select the **Close & Apply** to save your settings.
+4. **Use the SAS Token to setup permission for the power BI dashboards:** Generate the Shared Access Signature (SAS) token for storage account ingestion container to provide required permissions for the power BI dashboards. Follow the steps below:
+   - Navigate the FinOps deployment storage account, within the storage account blade under security + networking tab on the left, select **Shared access signature**. 
+   - On the Shared Accesss Signature blade. On the Allowed resource types select `Container` and `Object`, in the allowed permissions field select the **Read, List**, provide the start and expiry date and time range, keep the rest as default. Select `Generate SAS token and URL` button. Copy the generated token.
+  ![Image representing the SAS token configuration](../../assets/images/aoe/azure-storage-account-SAS.png)
+   - On the opened power BI dashboard, within **Transform data** window, select the `Data Source Settings`, within the windows select the `Edit permissions` button, in the edit permissions windows, click on 'Edit' button below credentials. 
+  ![data source settings within Transform Data](../../assets/images/aoe/powerbi-dashboard-SAS-setup.png)  
+   -  Select the Shared access signature tab and paste the copied Blob SAS token from Azure Portal. Click `Save` and 'Close' and then `Apply and Close' on the top ribbon of the transform window to apply the permissions and fetch the data.  
+  ![](../../assets/images/aoe/powerbi-dashboard-SAS-token.png)
+5. Select the **Close & Apply** to save your settings.
 
 If you run into any issues syncing your data, see [Troubleshooting Power BI reports](../../_resources/troubleshooting.md).
 
 <br>
 
+<!-- ## Setup FinOps dashboard with SAS token in power BI
```

Comment:
Is this a leftover from a previous thought about using a separate section for this? I wonder if that might actually be better. I like the idea of keeping the main instructions minimal while also providing the additional details for those looking to connect via a SAS token. Any reason you decided against this being a separate section?

Reply:
Hi @flanakin I was thinking to have it as a separate section, but the steps were in continuation to the initial setup documentation. if you insist, I can create a separate section for using SAS token for configuring Power BI permissions.

Reply:
I like your initial instincts to keep it separate. That should allow us to keep the initial instructions simple and also give us some more flexibility, if needed. Unless of course you have a concern with this approach that I may be missing.